scripts/mpv-clean

```diff
@@ -3,6 +3,7 @@ test -e mpv || exit 0
 
 cd mpv
 python3 ./waf distclean
+rm -rf build
```

Comment:
I don't get this change, and I don't see it explained.

Is this desirable regardless of meson? only with meson? Do we still need `waf clean` when using meson? etc.

Reply:
It's to unconditionally remove the build directory when using meson. waf distclean still executes (because waf is always downloaded), but it doesn't actually do anything if you didn't build with waf.

Reply:
IMO either make it an if/else where each mode only does what it needs, or explain it in a comment.

Reply:
Running both waf and meson clean makes sense to _really_ get rid of build artifacts when users switch between meson / waf when rebuilding.

Reply:
On that note, the rm -f line under this is very dated and doesn't do anything. The waf build puts all the generated files within the build directory. I'll remove that too since it's just confusing.